Line/iOS/LINELoginDemo/Podfile

```diff
@@ -0,0 +1,11 @@
+# Uncomment this line to define a global platform for your project
+# platform :ios, '9.0'
+
+target 'LINELoginDemo' do
+  # Uncomment this line if you're using Swift or would like to use dynamic frameworks
+  # use_frameworks!
+
+  # Pods for LINELoginDemo
+  pod 'Firebase/Auth', '3.6.0'
```

Comment:
I've updated to Firebase 3.7.1. When I started writing the code, the latest public version was 3.6.0 :)

And I decided to lock the SDK version in Podfile because I want to make sure that the developer will be able to download and run the sample code even when there's a breaking change in future Firebase iOS SDK release.